client/app/services/designer-state.service.js

```diff
@@ -141,12 +105,19 @@
 
       function matchAtomicServiceItemToTabs(srvTemplate) {
         var subTab;
-        var newItem;
         if (srvTemplate.service_type === 'atomic') {
           if (srvTemplate.prov_type !== 'generic') {
             subTab = findSubTabByProvType(srvTemplate.prov_type);
-          } else {
+            if (!subTab) {
+              console.log("Couldn't find toolbox Tab for Prov_Type: " + srvTemplate.prov_type);
```

Comment:
any chance we could use $log here? well here and everywhere else `console.log` was employed..   eslint will throwup otherwise, will be something we have to go back and change